etc/sync-function-template.js

```diff
@@ -476,7 +476,7 @@ function synctos(doc, oldDoc) {
     var itemValue = currentItemEntry.itemValue;
     var oldItemValue = currentItemEntry.oldItemValue;
 
-    if (typeof itemValue !== 'object') {
+    if (typeof itemValue !== 'object' || itemValue instanceof Array) {
```

Comment:
Addresses https://github.com/Kashoo/synctos/issues/42 for the "hashtable" validation type.